src/en/users-workflows.md

```diff
@@ -0,0 +1,94 @@
+Title: Juju workflow scenarios
+TODO: 
+
+
+# Workflow scenarios
+
+To demonstrate multi-user interaction, we present several workflows. It is
+assumed that Juju is installed [link to juju install page]. This presentation
+will be based on the local cloud: LXD. Since LXD does not require credentials,
+there will be no mention of credentials below (see
+[Getting Started](./getting-started.html) for a full treatment of using LXD as
+the cloud provider).
+
+
+## Basic setup and single user
+
+**Controller creation**<br/>
```

Comment:
There is a pretty solid case for the default landing page for docs doing whatever it takes to make it clear, not so much just this page. Anyhow, IIRC, you went and took them all out, so...